app/renderer/components/common/messageBox.js

```diff
@@ -3,20 +3,34 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const React = require('react')
-const ImmutableComponent = require('../immutableComponent')
+const {StyleSheet, css} = require('aphrodite')
```

Comment:
My solution in mind is:

![cascading](https://cloud.githubusercontent.com/assets/3362943/25553065/cecfed60-2ce4-11e7-88fa-1306ff9cd882.gif)

I think it might involve to send a patch to Aphrodite, still it seems worth implementing. With that improvement it would be easy to work with common components, enhancing them with more specific ones.

If that kind of cascading works only by improving `cx`, that would be quite fantastic! @cezaraugusto do you think it is possible?